connector-builder-server/src/main/openapi/openapi.yaml

```diff
@@ -57,6 +57,20 @@ paths:
           $ref: "#/components/responses/ExceptionResponse"
         "422":
           $ref: "#/components/responses/InvalidInputResponse"
+  /v1/manifest_template:
+    get:
+      summary: Return a connector manifest template to use as the default value for the yaml editor
+      operationId: template
+      responses:
+        "200":
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: string
+                description: Connector manifest template string
+        "400":
```

Comment:
We don't specify any parameter or body inputs, so would it be possible to get a 400 in this case? Or is this just convention to include at least some sort of error code?

Reply:
Good question -- I think I included it more out of habit but I agree that a 400 error probably wouldn't make sense for this endpoint. I think the only error that may happen here is something like an internal 500 server error, but according to various SO posts (e.g. [this](https://stackoverflow.com/questions/54238746/should-internal-error-500-be-part-of-swagger-api-documentation)) those should not be part of the OpenAPI documentation. So I'll just go ahead and remove this